doc-Managing_Authentication/topics/LDAP.adoc

```diff
@@ -55,15 +54,21 @@ In both LDAP and LDAPS, you can use groups from your directory service to set th
 * For LDAP users not belonging to a group:
 ** Select a {product-title_short} group from the *Default Group for Users* list. This default group can be used for all LDAP users who use LDAP for authentication only. Do not select *Get User Groups from LDAP*, which will hide the *Default Group for Users* option.
 * For LDAP users belonging to a group:
-** Check *Get User Groups from LDAP* to retrieve the user's group membership from LDAP. This is used for mapping a user's authorization to a {product-title_short} role. This requires group names on the LDAP server to match {product-title_short} group names.
+** Check *Get User Groups from LDAP* to retrieve the user's group membership from LDAP. This looks up all groups in the directory server for the user attempting to log in. This group list is matched against the available groups configured in {product-title_short}. Once a match is found, the role associated with the matching group identifies the authority the user will have on {product-title_short}. This requires group names on the directory server to match {product-title_short} group names. Checking this box enables user records to be automatically created in {product-title_short} when a user logs in.
```

Comment:
Per IBM style guide - Use "select" and "clear" for check boxes.

Reply:
👍

Reply:
Thanks for catching that - fixed 👍